drbert/data/dataset_readers.py

```diff
@@ -94,3 +104,16 @@ class SequenceLabellingDatasetReader(DatasetReader):
 
 class NLIDatasetReader(DatasetReader):
     pass
+
+if __name__ == "__main__":
+   
+    # Params for Cohort Task
+    path = None
+    partitions = None
+    tokenizer = AutoTokenizer.from_pretrained('bert-base-uncased', do_lower_case=True)
+    label_vocab = COHORT_DISEASE_CONSTANTS
+    batch_sizes = None
+
+    ICD_iters = DocumentClassificationDatasetReader(path, partitions, tokenizer, label_vocab, batch_sizes=batch_sizes, lower_case=True)
+
+    print(ICD_iters.text_to_iterator())
```

Comment:
This looks like it was used locally for testing? We should remove it from the repo